lib/kb_phylogenomics/kb_phylogenomicsImpl.py

```diff
@@ -3375,7 +3395,9 @@ def view_fxn_profile_phylo(self, ctx, params):
                         #if f_cnt % 100 == 0:
                         #    self.log (console, "prot: "+str(feature['protein_translation']))  # DEBUG
 
-                        if 'function' in feature and feature['function'] != None and feature['function'] != '':
+                        if 'function' in feature and feature['function'] != None and feature['function'] != '' \
```

Comment:
You can write this line more concisely as `if feature.get('function') or feature.get('functions')`.

`.get()` returns `None` for missing keys (or provide a second argument to supply the default value) and `None`, `""`, and `[]` are falsey